config/locales/actions.en.yml

```diff
@@ -0,0 +1,8 @@
+en:
+  actions:
+    archive_prompt:
+      label: "Archive"
+      confirmation: "Are you sure you want to archive %{name}?"
```

Comment:
this method is basically always going to receive an arbitrary string.

```
@user.full_name
"dong"
```

are both arbitrary strings as far as this method is concerned